### DISCUSSION TITLE
[SOLVED] Sensor & Cannula age stopped working after updating to Nightscout 14.2.x

### DISCUSSION THREAD

**Discussion by many7695** [nielsmaerten/gluco-check#80] [General]
When Gluco check is speaking the response to the age of sensor or cannula Age,  is “none years ago” the response to everything else is still correct

**Comment by nielsmaerten** [comment]
~~Have you updated your Nightscout to 14.2.x yet? This might be a bug with the latest version.  
I'm a little busy next week, but I'm going to look into this ASAP~~

It should be working again now 🎉

**Reply by many7695** [reply]
It is working again
